unit_tests/testSearch/testSearch.f90

```diff
@@ -14,11 +14,11 @@ PROGRAM testSearch
 
 IMPLICIT NONE
 
-INTEGER(SIK),DIMENSION(11) :: test_sorted = (/1,8,12,17,32,32,32,32,32,88,89/)
-INTEGER(SIK),DIMENSION(20) :: test_random = &
+INTEGER(SIK),DIMENSION(11) :: increasing_i = (/1,8,12,17,32,32,32,32,32,88,89/)
```

Comment:
I'd like to see some tests for decreasing as well

Reply:
Jesse pointed out that there is intention to rework a couple of these routines later (specifically for the decreasing case), and that it's indirectly tested via testInterpolators, so we won't worry about this for now